src/core/store/modules/collateral/collateral.reducer.ts

```diff
@@ -105,6 +135,35 @@ const collateralReducer = createReducer(collateralInitialState, (builder) => {
 
     // State changes from non collateral actions
 
+    /* -------------------------------- getLinePage ------------------------------- */
+    .addCase(
+      deploySecuredLineWithConfig.fulfilled,
+      (state, { payload: { lineAddress, escrowId, spigotId, deployData } }) => {
+        if (!lineAddress || !escrowId || !spigotId) return;
+        const { cratio } = deployData;
+        let map: CollateralMap = {};
+        const escrow = {
+          id: escrowId,
+          cratio: '0',
+          collateralValue: '0',
+          minCRatio: Number(cratio.toString()),
+          deposits: {} as EscrowDepositMap,
+          type: COLLATERAL_TYPE_ASSET,
+          line: lineAddress,
+        };
+        const spigot = {
+          id: spigotId,
+          revenueValue: '0',
+          revenueSummary: {} as RevenueSummaryMap,
+          type: COLLATERAL_TYPE_REVENUE,
+          line: lineAddress,
+        };
+        map[escrowId] = escrow;
+        map[spigotId] = spigot;
+        state.collateralMap = { ...state.collateralMap, ...map };
```

Comment:
would prefer u do `state.collateralMap = { ...state.collateralMap, spigotId: spigot, escrowId: escrow,  };` instead of mutating and having two destructurings

Reply:
done